ui/src/components/VolumeListTable.js

```diff
@@ -289,17 +263,86 @@ function Table({
     ?.isSortedDesc;
   useTableSortURLSync(sorted, desc, data);
 
+  const RenderRow = React.useCallback(
```

Comment:
To lighten your component, you could move `RenderRow` to its own component.

Reply:
Hello @nicolas2bert, 

Thanks for taking a look at this PR!
Just to be clear, you suggested me to move `RenderRow` to `VolumeListTable` component in my case?

Reply:
I suggest you move `RenderRow` component out of `VolumeListTable` for clarity purposes.

Reply:
OK. Actually `RenderRow` is a part of the `Table` component. And I render `Table` component in the `VolumeListTable`.

I guess the confusing part is I put the `Table` component inside `VolumeListTable` file, which should be separated.

Reply:
Yes, moving `Table` component out of `VolumeListTable` would be another refacto.
I was talking about moving `RenderRow` out from `Table` and so being its own component.

Reply:
The idea is that the component is already pretty heavy (and might need refacto later as you mention), let's try not to add too much code on top of it.

Reply:
TableRow could even live in its own file to reduce the size of VolumeListTable.js

Reply:
Indeed, thanks!